### PR TITLE
missing federation_resolve_endpoint in dumps

### PR DESCRIPTION
**Pull request by carlotafuro**
refer to issue federation resolve endpoint #279

### CODE REVIEW

File: examples/federation_authority/dumps/example.json [hunk 1]
```diff
@@ -193,6 +193,9 @@
     "trust_marks_issuers": {},
     "entity_type": "openid_relying_party",
     "metadata": {
+      "federation_entity": {
+        "federation_resolve_endpoint": "http://127.0.0.1:8000/oidc/rp/resolve"
```
Comment (peppelinux) [inline examples/federation_authority/dumps/example.json:197]: can I ask you to put also in federation_entity the following members?

organization_name
homepage_uri
policy_uri
logo_uri
contacts

File: examples/federation_authority/dumps/example.json [hunk 2]
```diff
@@ -635,6 +641,9 @@
         ]
       },
       "metadata": {
+        "federation_entity": {
```
Comment (peppelinux) [inline examples/federation_authority/dumps/example.json:644]: can I ask you to put also in federation_entity the following members?

organization_name
homepage_uri
policy_uri
logo_uri
contacts

File: examples/federation_authority/dumps/example.json [hunk 3]
```diff
@@ -1051,6 +1060,9 @@
...
```
Comment (peppelinux) [inline examples/federation_authority/dumps/example.json:1063]: can I ask you to put also in federation_entity the following members?

organization_name
homepage_uri
policy_uri
logo_uri
contacts

File: examples/federation_authority/dumps/example.json [hunk 4]
```diff
@@ -1410,6 +1422,9 @@
       }
     ],
     "metadata": {
+      "federation_entity": {
```
Comment (peppelinux) [inline examples/federation_authority/dumps/example.json:1425]: can I ask you to put also in federation_entity the following members?

organization_name
homepage_uri
policy_uri
logo_uri
contacts

File: examples/federation_authority/dumps/example.json [hunk 5]
```diff
@@ -1663,6 +1678,9 @@
         ]
     },
     "metadata": {
+      "federation_entity": {
```
Comment (peppelinux) [inline examples/federation_authority/dumps/example.json:1681]: can I ask you to put also in federation_entity the following members?

organization_name
homepage_uri
policy_uri
logo_uri
contacts

File: examples/provider/dumps/example.json [hunk 6]
```diff
@@ -94,6 +94,9 @@
     "trust_marks_issuers": {},
     "entity_type": "openid_provider",
     "metadata": {
+      "federation_entity": {
+        "federation_resolve_endpoint": "http://127.0.0.1:8002/oidc/op/resolve"
```
Comment (peppelinux) [inline examples/provider/dumps/example.json:98]: can I ask you to put also in federation_entity the following members?

organization_name
homepage_uri
policy_uri
logo_uri
contacts

File: examples/relying_party/dumps/example.json [hunk 7]
```diff
@@ -63,6 +63,9 @@
     "trust_marks_issuers": {},
     "entity_type": "openid_relying_party",
     "metadata": {
+      "federation_entity": {
+        "federation_resolve_endpoint": "http://127.0.0.1:8001/resolve"
```
Comment (peppelinux) [inline examples/relying_party/dumps/example.json:67]: can I ask you to put also in federation_entity the following members?

organization_name
homepage_uri
policy_uri
logo_uri
contacts

File: examples/relying_party/dumps/example.json [hunk 8]
```diff
@@ -193,6 +196,9 @@
         ]
       },
       "metadata": {
+        "federation_entity": {
+          "federation_resolve_endpoint": "http://127.0.0.1:8000/oidc/op/resolve"
```
Comment (peppelinux) [inline examples/relying_party/dumps/example.json:200]: can I ask you to put also in federation_entity the following members?

organization_name
homepage_uri
policy_uri
logo_uri
contacts

File: examples/relying_party/dumps/example.json [hunk 9]
```diff
@@ -477,6 +483,9 @@
         ]
       },
       "metadata": {
+        "federation_entity": {
+          "federation_resolve_endpoint": "http://127.0.0.1:8002/oidc/op/resolve"
```
Comment (peppelinux) [inline examples/relying_party/dumps/example.json:487]: can I ask you to put also in federation_entity the following members?

organization_name
homepage_uri
policy_uri
logo_uri
contacts

File: examples/relying_party/dumps/example.json [hunk 10]
```diff
@@ -763,6 +772,9 @@
       }
     ],
     "metadata": {
+      "federation_entity": {
+        "federation_resolve_endpoint": "http://127.0.0.1:8000/oidc/op/resolve"
```
Comment (peppelinux) [inline examples/relying_party/dumps/example.json:776]: can I ask you to put also in federation_entity the following members?

organization_name
homepage_uri
policy_uri
logo_uri
contacts

File: examples/relying_party/dumps/example.json [hunk 11]
```diff
@@ -942,6 +954,9 @@
         ]
     },
     "metadata": {
+      "federation_entity": {
+        "federation_resolve_endpoint": "http://127.0.0.1:8002/oidc/op/resolve"
```
Comment (peppelinux) [inline examples/relying_party/dumps/example.json:958]: can I ask you to put also in federation_entity the following members?

organization_name
homepage_uri
policy_uri
logo_uri
contacts

**Review by peppelinux — CHANGES_REQUESTED** [review]

File: examples/federation_authority/dumps/example.json (hunk 1 above)
Comment (carlotafuro) [reply examples/federation_authority/dumps/example.json]: ok!

File: examples/federation_authority/dumps/example.json (hunk 2 above)
Comment (carlotafuro) [reply examples/federation_authority/dumps/example.json:662]: ok!

File: examples/federation_authority/dumps/example.json (hunk 3 above)
Comment (carlotafuro) [reply examples/federation_authority/dumps/example.json:1088]: ok!

File: examples/federation_authority/dumps/example.json (hunk 4 above)
Comment (carlotafuro) [reply examples/federation_authority/dumps/example.json:1457]: ok!

File: examples/relying_party/dumps/example.json (hunk 8 above)
Comment (carlotafuro) [reply examples/relying_party/dumps/example.json]: ok!

File: examples/relying_party/dumps/example.json (hunk 9 above)
Comment (carlotafuro) [reply examples/relying_party/dumps/example.json]: ok!

File: examples/relying_party/dumps/example.json (hunk 10 above)
Comment (carlotafuro) [reply examples/relying_party/dumps/example.json]: ok!

File: examples/relying_party/dumps/example.json (hunk 11 above)
Comment (carlotafuro) [reply examples/relying_party/dumps/example.json]: ok!

**Review by peppelinux — APPROVED** [review]

**Pull request merged**